apps/parktime/park_time.star

```diff
@@ -0,0 +1,339 @@
+        render.Text(content = "GROUPS", font = "CG-pixel-3x5-mono", height = 6, color = DEFAULT_COLOR_LIGHT_GRAY),
+        render.Box(width = 64, height = 1),
+        render.Text(content = wait_time, font = "5x8", height = 9, color = "#FFFFFF"),
+        render.Box(width = 64, height = 2),
+    ]
+
+def wait_times(entity_live, display_name, color):
+    wait_time_label = "WAIT TIME"
+    wait_time = ""
+    wait_time_units_color = DEFAULT_COLOR_LIGHT_GRAY
+    if entity_live["status"] == "CLOSED":
+        wait_time_units = "CLOSED"
+        wait_time_units_color = DEFAULT_COLOR_DANGER
+    elif entity_live["status"] == "DOWN":
+        wait_time_units = "TEMP CLOSED"
+        wait_time_units_color = DEFAULT_COLOR_WARNING
+    elif "queue" in entity_live:
+        if entity_live["queue"]["STANDBY"]["waitTime"]:
+            minutes = str(int(entity_live["queue"]["STANDBY"]["waitTime"]))
+            wait_time = minutes
+        else:
+            wait_time = "0"
+        wait_time_units = "MINUTES"
+    else:
+        wait_time_units = "CLOSED"
+        wait_time_units_color = DEFAULT_COLOR_DANGER
+    return [
+        render.Marquee(width = 64, align = "center", child = render.Text(content = display_name, font = "5x8", color = color)),
+        render.Text(content = wait_time_label, font = "CG-pixel-3x5-mono", height = 6, color = DEFAULT_COLOR_LIGHT_GRAY),
+        render.Box(width = 64, height = 1),
+        render.Box(
+            width = 17,
+            height = 9,
+            color = DEFAULT_COLOR_DARK_GRAY,
+            padding = 1,
+            child = render.Box(
+                width = 15,
+                color = DEFAULT_COLOR_BLACK,
+                child = render.Padding(child = render.Text(content = wait_time, font = "CG-pixel-4x5-mono", color = "#FFFFFF"), pad = (2, 1, 1, 1)),
+            ),
+        ),
+        render.Text(content = wait_time_units, font = "CG-pixel-3x5-mono", height = 6, color = wait_time_units_color),
+        render.Box(width = 64, height = 2),
+    ]
+
+def no_data(display_name, color):
+    return [
+        render.Marquee(width = 64, align = "center", child = render.Text(content = display_name, font = "5x8", color = color)),
+        render.Box(width = 64, height = 8),
+        render.Text(content = "NO DATA", font = "5x8", color = "#FF0000"),
+        render.Box(width = 64, height = 8),
+    ]
+
+def main(config):
+    display_widgets = []
+    color = DEFAULT_COLOR
+    timezone = config.get("$tz", "America/New_York")  # Utilize special timezone variable
+    today = time.now().in_location(timezone).format("2006-01-02")
+    tomorrow = (time.now().in_location(timezone) + time.parse_duration("24h")).format("2006-01-02")
+
+    # Get entity if specified
+    entity = DEFAULT_ENTITY
+    if config.get("entity"):
+        entity = config.get("entity")
+
+    #Get the entity info
+    res = http.get(THEME_PARKS_WIKI_URL + entity)
```

Comment:
Is this call redundant? Noticed the result isn't really used anywhere.

Reply:
If not, should it also have a configured cache TTL?

Reply:
yes, thank you. I will remove the redundancy.